Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileCleaner
+    {
+        private readonly string _dirtyFile;
+        private readonly string _cleanedFile;
+
+        public FileCleaner(string dirtyFile, string cleanedFile)
+        {
+            _dirtyFile = dirtyFile;
+            _cleanedFile = cleanedFile;
+        }
+
+        public void Clean()
+        {
+            if (!Test.IsStringValid(_dirtyFile))
+            {
+                throw new ArgumentException("Invalid file path!", _dirtyFile);
+            }
+            if (!Test.IsStringValid(_cleanedFile))
+            {
+                throw new ArgumentException("Invalid file path!", _cleanedFile);
+            }
+
+            string dirtyData = File.ReadAllText(_dirtyFile);
+            if (!Test.IsStringValid(dirtyData))
+            {
+                File.WriteAllText(_cleanedFile, dirtyData);
+            }
+
+            string cleanedData = dirtyData.Replace(Settings.corruptionChar, Settings.emptyChar);
+            string testedData = TestData(cleanedData);
+            File.WriteAllText(_cleanedFile, testedData);
+        }
+
+        private string TestData(string cleanData)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'TestData' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)